setup.py

```diff
@@ -26,8 +26,8 @@
     url='https://github.com/boykoc/ckanext-ontario_theme',
```

Comment:
`version='0.0.1'` on line 20 should be corrected to the versioning scheme we are using. I would like a comment on this from @bianca 
current release v1.0 according to tags (https://github.com/ongov/ckanext-ontario_theme/releases)

Reply:
update to 2.0.0, please.